cannot compile under Centos 7

when running `make` under Centos 7, I got this error:

```
src/netlink.c: In function 'netlink_link_add_bridge':
src/netlink.c:254:25: error: 'IFLA_BR_MCAST_SNOOPING' undeclared (first use in this function)
    mnl_attr_put_u8(nlh, IFLA_BR_MCAST_SNOOPING, 0);
                         ^~~~~~~~~~~~~~~~~~~~~~
src/netlink.c:254:25: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [src/cc_oci_runtime-netlink.o] Error 1
make[1]: Leaving directory `/home/cloud/go/src/github/01org/cc-oci-runtime'
make: *** [all] Error 2
```
Centos 7 kernel version:
```
[cloud@centos7-dm cc-oci-runtime]$ uname -r
3.10.0-514.10.2.el7.x86_64 
```

Related to https://github.com/01org/cc-oci-runtime/commit/001e2648f540ed7672ae67c7f3c369120a0c25fc

--- a/src/netlink.c
+++ b/src/netlink.c
@@ -39,6 +39,13 @@
 #include "util.h"
 #include "networking.h"
 
+/* We need to define this as this enum may not be present
+ * in older kernels. There isn't a way to check for its presence
+ * as the definition associated (IFLA_BR_MAX) with this enum
+ * is not available as preprocessor time.
+ */
+#define IFLA_BR_MCAST_SNOOPING 23
+
 /*!
  * Setup the netlink socket to use with netlink
  * transactions.
@@ -246,6 +253,14 @@ netlink_link_add_bridge(struct netlink_handle *const hndl,
 	 */
 	#ifdef IFLA_BR_MAX
 		if (IFLA_BR_MAX > 23) {
+			/* Undefining this macro, to use the kernel
+			 * definition instead. This is not required, as the
+			 * numeric value associated with the kernel enum element will
+			 * not change, but always better to use the kernel enum
+			 * instead.
+			 */
+			#undef IFLA_BR_MCAST_SNOOPING
+
 			disable_snooping = true;
 			g_debug("Turning off multicast snooping for bridge %s",
 				name);

